src/main/java/com/epam/aidial/core/storage/BlobStorage.java

```diff
@@ -254,6 +256,20 @@ private static ContentMetadata buildContentMetadata(String contentType) {
         return BaseMutableContentMetadata.fromContentMetadata(contentMetadata);
     }
 
+    private static CredentialProvider getCredentialProvider(StorageProvider provider, String identity, String credential) {
+        return switch (provider) {
```

Comment:
I think we can replace `switch` a simple `if/else`

Reply:
I would leave as is, if you provider added - you will face compilation error